### PR TITLE
add support for url params in Checks.List() + tags

### PR DESCRIPTION
**Pull request by jamesonjlee**
https://www.pingdom.com/resources/api#MethodGet+Check+List

possible to extend this to Contacts.List() to allow using `limit` and `offset`

### CODE REVIEW

File: pingdom/api_responses.go [hunk 1]
```diff
@@ -19,26 +19,33 @@ type PingdomError struct {
...
+	Paused                   bool               `json:"paused,omitempty"`
+	ContactIds               []int              `json:"contactids,omitempty"`
+	Type                     CheckResponseType  `json:"type,omitempty"`
+	Tags                     []CheckResponseTag `json:"tags,omitempty"`
+}
+
+type CheckResponseTag struct {
+	Name  string `json:"name"`
+	Type  string `json:"type"`
+	Count int    `json:"count"`
```
Comment (jamesonjlee) [inline pingdom/api_responses.go]: this actually has to be interface{} sometime count is retruened as a string.

Comment (cryptobioz) [reply pingdom/api_responses.go]: According to the documentation, on might think that the API returns an int but, in fact, it returns a string.
Maybe we could use a type string for this field?

Comment (partamonov) [reply pingdom/api_responses.go]: Just used that and it return `int` for me for api/2.1

**Comment by russellcardullo** [comment]
Merged in a new PR since the fork no longer exists in #16

**Pull request closed without merging**